meta-jsdelivr/recipes-jsdelivr/jsdelivr-scripts/files/jsdelivr-startWorld.sh

```
@@ -1,50 +1,183 @@
+
+# Check if partition exists before mounting
+if [ ! -b "$DOCKER_PARTITION" ]; then
+    echo "ERROR: Docker storage partition $DOCKER_PARTITION does not exist!" > /dev/tty3
+    echo "ERROR: Docker storage partition $DOCKER_PARTITION does not exist!" > /dev/console
+    # Boot failed - set solid RED
+    led_boot_failed
+    # Continue without mounting - Docker will use tmpfs
+else
+    mkdir -p /var/lib/docker
+    /bin/mount -o noatime "$DOCKER_PARTITION" /var/lib/docker
+    if [ $? -ne 0 ]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)